blog/models.py

```diff
@@ -65,92 +70,64 @@ class Ministrant(models.Model):
     unicode_name = models.CharField(max_length=100, blank=False)
 
     def __str__(self) -> str:
-        """
+        '''
         Returns a string representation of the object containing the surname and birth name.
-
-        :return: A string representing the object.
-        """
+        
+        Returns
+        -------
+        str
+            A string representation of the object containing the surname and birth name.
+        '''
         return f'{self.surname} {self.birthname}'
-
+    
     def get_absolute_url(self) -> str:
-        """
-        Returns the absolute URL to access the detail view of a `Ministrant` instance.
-
-        :param self: The `Ministrant` instance to generate the URL for.
-        :type self: Ministrant
-        :return: The absolute URL to access the detail view of the `Ministrant` instance.
-        :rtype: str
-        """
+        '''
+        Returns the absolute URL of the object.
```

Comment:
Docstrings should be written in imperative mood. Docstring should describe the function effect as a command.
https://peps.python.org/pep-0257/
https://docs.astral.sh/ruff/rules/non-imperative-mood/